### DISCUSSION TITLE
Copilot Monthly Digest 🗞️

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#68304] [Copilot News and Announcements]
👋 Welcome to the September edition of Copilot Monthly Roundup. Let's take a brief look at the recent updates in the Copilot product world 🌏. 

🔢 Let’s Go!

## [GitHub Copilot Chat beta now available for all individuals](https://github.blog/2023-09-20-github-copilot-chat-beta-now-available-for-all-individuals/) 🆕 

![Screenshot 2023-09-21 at 2 45 52 PM](https://github.com/community/community/assets/95834130/383168d0-fc52-4c17-a5eb-19fa52fbc1ea)

**GitHub Copilot Chat is now available free in public beta as part of the GitHub Copilot for individuals subscription** 🎉 .

Integrated together, GitHub Copilot Chat and the GitHub Copilot pair-programmer form a new, powerful AI-assistant allowing every developer to build in the natural language of their choice.

GitHub Copilot Chat is supported in both Visual Studio and Visual Studio Code editors. **GitHub Copilot for Individual users will also receive an email notification to guide them on the next steps**. If you’re not already part of our beta program and would like to get started, please refer to our comprehensive [getting started guide](https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat), which is conveniently linked in your email notification.

For more details, please read the [blog post](https://github.blog/2023-09-20-github-copilot-chat-beta-now-available-for-all-individuals/). Please note you must have a GitHub Copilot for individuals subscription to enroll.

## [Improved Copilot for Business Admin settings experience and audit log integration](https://github.blog/changelog/2023-09-07-github-copilot-september-7th-update/) 💼 

![Screenshot 2023-09-12 at 6 44 15 PM](https://github.com/community/community/assets/95834130/2fe60c95-4d5e-43ab-994f-7436750dbada)

This past month we introduced a Revamped Copilot for Business Admin Experience. Let’s check out what’s new:

- 🔑 **License Management**: Easily manage licenses and monitor your team’s impact on your bill.
- 👀 **Audit Log Integration**: Keep a close watch on who's making changes with the new GitHub Audit Log integration.
- 💼 **Streamlined Seat Purchasing**: We've simplified seat purchasing. Review Copilot seats and charges upfront, update assignments effortlessly, and see bill changes in real-time when adding or removing users or teams.
- ⚙️ **Troubleshooting Made Easy**: Admins can now troubleshoot issues within their organizations seamlessly. No need to contact support – just use the GitHub Audit Log's "Copilot Activity" filter to track settings changes, policy updates, and seat changes.

Your Copilot admin experience is now more intuitive and efficient, helping you stay in control.

## [Copilot Chat Updates](https://github.blog/changelog/2023-08-28-github-copilot-august-28th-update/)

https://github.com/community/community/assets/95834130/3b040eb3-9d78-48b3-8025-f653b7a82e7d

- ⌛**Save time with slash command improvements** : Forget deleting commands one letter at a time. With the latest update, commands now render as blocks and can be deleted with a single backspace! We’ll also automatically execute slash commands like `/clear` and `/help` when they’re selected from the suggestion list, removing the need to explicitly submit the chat request after accepting a completion.
- ⚡ **Updated Quick Chat experience**: Prefer an ephemeral chat experience instead of a panel or in-editor one? Quickly activate this view with `Shift + Cmd/Ctrl + I`. In this update, we’ve added conversation history and support for slash command completion to bring it closer to our other chat experiences.
- 🆕 **Expanded context to include the terminal**: Copilot chat context now extends beyond code files, taking in the active terminal’s buffer and selection to better inform its responses.
- 🆙 **Expanding Copilot’s context window to 8k**: We’ve officially rolled out the 8k context window for all code completion requests! 🥳 With this change, Copilot has greater flexibility to include additional information as part of requests and ultimately improve the suggestions you receive!

That’s not it, we’ve also shipped an all new context view in Copilot Chat. 👇👇

## [Interactive Code Assistant View in Visual Studio](https://github.blog/changelog/2023-08-28-github-copilot-august-28th-update/#stay-in-the-flow-with-interactive-code-assistant-view-in-visual-studio) 🤝 

This past month, GitHub Copilot Chat got updated with the new Interactive Code Assistant View in Visual Studio. You can now:
- 👌 Refine code effortlessly: Chat with Copilot directly in your editor—no need to switch to a chat window.
- ❓ Ask questions with "Ask Copilot" in your code and get inline answers.
- 🔎 Review, correct, and refine suggestions alongside your code in Visual Studio's familiar diff view.
- 💪 Stay in control throughout the process.

[Learn more about this feature in our latest blog](https://devblogs.microsoft.com/visualstudio/simplified-code-refinement-and-debugging-with-github-copilot-chat/).

## Trending in the Community 📊

- [Copilot ads in the Code Viewer](https://github.com/orgs/community/discussions/65073) - We are in the process of revamping this experience, so if you noticed a change — We're looking into it!
- [GitHub Copilot for VSCode stopped working](https://github.com/orgs/community/discussions/65894) - We noticed that users with outdated versions of VSCode were not able to use Copilot. Always keep your IDE [up-to-date](https://learn.microsoft.com/en-us/visualstudio/install/update-visual-studio?view=vs-2022) to avoid situations like these.
- [Error: GitHub Copilot could not connect to server. Extension activation failed](https://github.com/orgs/community/discussions/64334) - This issue was caused by invalid hosts file and firewall settings. If you're still facing this this issue or ever face a similar one, check out the [Copilot network troubleshooting Docs](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-firewall-settings-for-github-copilot). 

## Learning Content

- ⚡  **[Getting Started with GitHub Copilot Chat in VSCode](https://github.com/orgs/community/discussions/64517)** - Learn more about the new chat functionality powered by GitHub Copilot. Read the step-by-step [getting started guide](https://github.com/orgs/community/discussions/64517) by [@LadyKerr](https://github.com/LadyKerr) and explore your leveled up 🍄  pair-programmer!
- 🏗️  **[How I used GitHub Copilot to build a browser extension](https://github.blog/2023-05-12-how-i-used-github-copilot-to-build-a-browser-extension/)** - Watch how @blackgirlbytes built a Chrome extension in their seven-step process and three major takeaways about learning and pair programming in the age of AI.
- 💯  **[How to write better prompt with GitHub Copilot](https://github.blog/2023-06-20-how-to-write-better-prompts-for-github-copilot/)** - Crafting effective prompts is an (sometimes frustrating) art. Here's how you can approach it with finesse: Be specific, aim for short outputs, and experiment. [Learn more tips](https://github.blog/2023-06-20-how-to-write-better-prompts-for-github-copilot/) for getting the output you want. 
- 💼  **[How to build an enterprise LLM application: Lessons from GitHub Copilot](https://github.blog/2023-09-06-how-to-build-an-enterprise-llm-application-lessons-from-github-copilot/)** - How did we take an idea for an AI coding assistant and turn it into GitHub Copilot? It took a well-defined problem, an entrepreneurial spirit, and a lot of feedback and iteration. If you also want to build and scale an LLM app, you'll want to take notes. [Find out more](https://github.blog/2023-09-06-how-to-build-an-enterprise-llm-application-lessons-from-github-copilot/) from the team behind GitHub Copilot. 🚀

**Comment by KasiaSun** [comment]
Love these updates! Thanks for sharing!
